setup.py

```diff
@@ -62,7 +62,7 @@
         "pydantic>=1.10.14",
         "litestar>=2.6.3",
         "typing-inspect>=0.9.0",
-        "uvicorn>=0.22.0",
+        "uvicorn[standard]>=0.22.0",
```

Comment:
This includes some dependencies which can greatly improve uvicorn's performance, such as uvloop and `httptools` instead of `h11`